orion-ui/src/pages/ConcurrencyLimit.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <p-layout-well class="concurrencyLimit">
+    <template #header>
+      <PageHeadingConcurrencyLimit v-if="concurrencyLimit" :concurrency-limit="concurrencyLimit" @delete="deleteConcurrencyLimit" />
+    </template>
+
+    <p-tabs :tabs="tabs">
+      <template #details>
+        <ConcurrencyLimitDetails v-if="concurrencyLimit" :concurrency-limit="concurrencyLimit" />
+      </template>
+      <template #active-task-runs>
+        <ConcurrencyLimitActiveRuns v-if="concurrencyLimit?.activeSlots" :active-slots="concurrencyLimit.activeSlots" />
+      </template>
+    </p-tabs>
+
+    <template #well>
+      <ConcurrencyLimitDetails v-if="concurrencyLimit" alternate :concurrency-limit="concurrencyLimit" />
+    </template>
+  </p-layout-well>
```

Comment:
If everything needs the concurrency limit before being displayed I'd just put a v-if on the layout component rather than adding it to each component on the page.